controllers/topics.js

```diff
@@ -16,4 +17,12 @@ exports.getArticles = (req, res, next) => {
     .then((articles) => {
         res.send(articles.rows)
     })
+}
+
+exports.getCommentsByArticleId = (req, res, next) => {
+    const { article_id } = req.params
+    selectCommentsByArticleId(article_id)
+    .then((comments) => {
+        res.send(comments.rows)
```

Comment:
`res.send({comments: comments.rows})` - from above comment

Reply:
On saying this though - it might be useful if your model simply resolves with the array of comments, rather than the result object and key of rows